__tests__/app.test.js

```diff
@@ -41,6 +42,72 @@ describe("Using app.js to run the database of NC-games", () => {
         });
     });
   });
-
-  //Head describe DETETE later
+  describe("4: GET/api/reviews/:review_id", () => {
+    test("200: responds with a single matching review", () => {
+      const review_id = 2;
+      return request(app)
+        .get(`/api/reviews/${review_id}`)
+        .expect(200)
+        .then((response) => {
+          expect(response.body.game).toHaveProperty(
+            "review_id",
+            expect.any(Number)
+          );
+          expect(response.body.game).toHaveProperty(
+            "title",
+            expect.any(String)
+          );
+          expect(response.body.game).toHaveProperty(
+            "review_body",
+            expect.any(String)
+          );
+          expect(response.body.game).toHaveProperty(
+            "designer",
+            expect.any(String)
+          );
+          expect(response.body.game).toHaveProperty(
+            "review_img_url",
+            expect.any(String)
+          );
+          expect(response.body.game).toHaveProperty(
+            "votes",
+            expect.any(Number)
+          );
+          expect(response.body.game).toHaveProperty(
+            "category",
+            expect.any(String)
+          );
+          expect(response.body.game).toHaveProperty(
+            "owner",
+            expect.any(String)
+          );
+          expect(response.body.game).toHaveProperty(
+            "created_at",
+            expect.any(String)
+          );
+        });
+    });
+    test("400: ? GET/api/review/:review_id where the id = 9999", () => {
+      const review_id = 9999;
+      return request(app)
+        .get(`/api/reviews/${review_id}`)
+        .expect(400)
+        .then((response) => {
+          expect(response.body).toEqual({
+            status: 400,
+            msg: "Bad Request",
+          });
+        });
+    });
+    test("404: Not Found from possible spelling mistake for reviews (safety netting)", () => {
```

Comment:
swap 400 and 404 here, /api/reviews/9999 is a valid path as the type of review_id is still a number, but review 9999 does not exist. Whereas a typo in the path is a 400 because it is a bad request, that endpoint does not exist (would suggest doing something like /api/reviews/bananas for this test case, then you would just need to check if your review_id is a number.

Reply:
Switch them over and added an addtional test for the other... basically there is now 404 and 400 error handling